classes/php.php

```diff
@@ -41,6 +41,11 @@ public function setBinaryPath($phpPath = null)
 			}
 		}
 
+		if (basename($phpPath) === 'hhvm')
```

Comment:
@pschultz I did not merge or rebase this branch because of no feedback: I do not use HHVM and I'm working blindly here. If you are ok to test it with the changes you requested, I'll fix it and rebase.